TV/AppDelegate.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
```

Comment:
Should we remove this line for the sake of perfection?

Reply:
Fixed!